SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -1044,6 +1044,45 @@ void USpatialNetDriver::ServerReplicateActors_ProcessPrioritizedActors(UNetConne
 	// SpatialGDK - Here Unreal would return the position of the last replicated actor in PriorityActors before the channel became saturated.
 	// In Spatial we use ActorReplicationRateLimit and EntityCreationRateLimit to limit replication so this return value is not relevant.
 }
+
+void USpatialNetDriver::ProcessRPC(AActor* Actor, UObject* SubObject, UFunction* Function, void* Parameters)
+{
+	// The RPC might have been called by an actor directly, or by a subobject on that actor
+	UObject* CallingObject = SubObject != nullptr ? SubObject : Actor;
+
+	if (IsServer())
+	{
+		// Creating channel to ensure that object will be resolvable
+		if (GetOrCreateSpatialActorChannel(CallingObject) == nullptr)
+		{
+			UE_LOG(LogSpatialOSNetDriver, Warning, TEXT("Failed to get or create SpatialActorChannel for %s"), *CallingObject->GetName());
+		}
+	}
+
+	int ReliableRPCIndex = 0;
```

Comment:
What is this for? Is this a holdover from when we were using commands.

Reply:
It's still used if we switch on bool bCheckRPCOrder;

Reply:
It shouldn't be. The order is the order they arrive in.

Reply:
However this means that if something gets dropped then we queue forever which would be bad in multi-worker, where delivery might not happen.

Reply:
I suspect that that functionality won't work anymore, so it's worth removing it completely. I don't think it should block this merge though.

Reply:
Created a jira ticket to get rid of it and redesign it.

Reply:
Why shouldn't we just kill it now? If it's just a stub in a couple of places.

Reply:
We should at least look to see what behaviour we expect. If we run out of time then deferring it is fine. But I'm not a fan of dealing with PR comments in a different PR unless they are unrelated or involve refactoring some tangential piece of code.

Reply:
I considered this as a tangential piece of code and so haven't touched it yet.